src/term/mod.rs

```diff
@@ -701,6 +705,28 @@ impl Term {
     })
   }
 
+  /// Returns whether any substitution happened within the term or not
```

Comment:
Without the previous context, this description is not sufficient, since it only talks about the return and not what it does to the term